.github/workflows/test.yml

```
@@ -0,0 +1,24 @@
+name: CI
```

how about `Test` to follow our naming convention
https://nimblehq.co/compass/development/code-conventions/github-actions/#naming
> Name workflow files following the pattern verb_environment.yml. environment can be omitted if the workflow does not vary per environment

```suggestion
name: Test
```